### PR TITLE
fix(KFLUXBUGS-1321): support internal use case

### PR DESCRIPTION
**Pull request by scoheb**
- support for production-internal and stage-internal

### CODE REVIEW

**Review by mmalina — DISMISSED** [review]

File: tasks/push-rpm-manifests-to-pyxis/push-rpm-manifests-to-pyxis.yaml [hunk 1]
```diff
@@ -110,8 +110,14 @@ spec:
         elif [[ "$(params.server)" == "stage" ]]
         then
           export PYXIS_GRAPHQL_API="https://graphql-pyxis.preprod.api.redhat.com/graphql/"
+        elif [[ "$(params.server)" == "production-internal" ]]
```
Comment (johnbieren) [inline tasks/push-rpm-manifests-to-pyxis/push-rpm-manifests-to-pyxis.yaml:113]: line 24 should be updated too (which is copied in the README)

Comment (scoheb) [reply tasks/push-rpm-manifests-to-pyxis/push-rpm-manifests-to-pyxis.yaml:113]: fixed

File: tasks/push-rpm-manifests-to-pyxis/README.md [hunk 2]
```diff
@@ -8,8 +8,11 @@ Tekton task that extracts all rpms from the sboms and pushes them to Pyxis as an
 |------|-------------|----------|---------------|
 | pyxisJsonPath | Path to the JSON string of the saved Pyxis data in the data workspace | No | - |
 | pyxisSecret | The kubernetes secret to use to authenticate to Pyxis. It needs to contain two keys: key and cert | No | - |
-| server | The server type to use. Options are 'production' and 'stage' | Yes | production |
+| server | The server type to use. Options are 'production','production-internal,'stage-internal' and 'stage'. | Yes | production |
 | concurrentLimit | The maximum number of images to be processed at once | Yes | 4 |
 
+## Changes in 0.1.2
+* add support for production-internal and stage-interal
```
Comment (johnbieren) [inline tasks/push-rpm-manifests-to-pyxis/README.md]: `s/stage-interal/stage-internal/`

**Review by johnbieren — DISMISSED** [review]

**Review by johnbieren — APPROVED** [review]

**Pull request merged**